packages/grafana-ui/src/components/DateTimePickers/TimeZonePicker.story.tsx

```diff
@@ -1,41 +1,47 @@
 import { action } from '@storybook/addon-actions';
-import { ComponentMeta } from '@storybook/react';
+import { useArgs } from '@storybook/client-api';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
 import React from 'react';
 
 import { TimeZonePicker } from '@grafana/ui';
 
-import { UseState } from '../../utils/storybook/UseState';
 import { withCenteredStory } from '../../utils/storybook/withCenteredStory';
 
 const meta: ComponentMeta<typeof TimeZonePicker> = {
   title: 'Pickers and Editors/TimePickers/TimeZonePicker',
   component: TimeZonePicker,
   decorators: [withCenteredStory],
+  parameters: {
+    controls: {
+      exclude: ['inputId', 'onChange', 'onBlur'],
+    },
+  },
+  args: {
+    value: 'Europe/Stockholm',
+  },
+  argTypes: {
+    includeInternal: {
+      control: {
+        type: 'boolean',
+      },
+    },
+  },
 };
 
-export const basic = () => {
+export const Basic: ComponentStory<typeof TimeZonePicker> = (args) => {
+  const [, updateArgs] = useArgs();
   return (
-    <UseState
-      initialState={{
-        value: 'Europe/Stockholm',
+    <TimeZonePicker
+      {...args}
+      onChange={(newValue) => {
+        if (!newValue) {
+          return;
+        }
```

Comment:
I know this was already there before, but this means we can't select "Default" because the value of that is `""` which evaluates to `true` here. I'm not sure why we do this check anyway, should we remove it?